xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -1840,14 +1840,16 @@ CacheInfo CVideoPlayer::GetCachingTimes()
   info.delay = cache_left - play_left;
   info.time = cache_time;
 
-  if (lowrate > 0)
+  if (lowrate > 0 || cacheIsFull)
```

Comment:
I pretty sure this is wrong. This means that every time the cache is full, which is a normal/valid condition, the block below is triggered. Mind explaining your reasoning?

Reply:
Just below this line, there's a comment:
```cpp
1847  // buffer is full & our read rate is too low
1848  CLog::Log(LOGDEBUG, "Readrate {} was too low with {} required", lowrate, maxrate);
```
Going by this comment's sentiment, `lowrate` actually means that we're not reading from it fast enough, which will be true all the times that the cache gets to full. If I remove this then the video will halt from the second time onwards when the cache gets to full and the problem remains as it is.